torchvision/io/_video_opt.py

```diff
@@ -14,6 +14,12 @@
 except (ImportError, OSError):
     _HAS_VIDEO_OPT = False
 
+try:
+    _load_library("Decoder")
+    _HAS_VIDEO_DECODER = True
+except (ImportError, OSError):
+    _HAS_VIDEO_DECODER = False
+
```

Comment:
nit: I would probably have moved this to the file where `GPUDecoder` is actually used.

I would actually in the future move all the code related to `VideoReader` to a separate file, and not keep it in `__init__.py`.

Reply:
https://github.com/pytorch/vision/issues/5148